recipes/acados/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "acados" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: a6868cf31b313702f186214812de3857a120731417a95faf1fed00ff17124172
+
+build:
+  number: 0
+  rpaths:
+    - lib/
+    - lib/{{ name }}/
+  run_exports:
+    - {{ pin_subpackage('acados', max_pin='x.x') }}
+
+requirements:
+  build:
+    - pkg-config
+    - cmake
+    - ninja
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - {{ pin_compatible('numpy') }}
```

In host it should be just ``numpy``, the pin_compatible in run is correct :)
```suggestion
    - numpy
```